src/components/pages/Bookings/BookingsRecapTable/CellsFormatter/__specs__/BookingDateCell.spec.jsx

```diff
@@ -1,25 +1,25 @@
-import { shallow } from 'enzyme/build'
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
 import React from 'react'
 
 import BookingDateCell from '../BookingDateCell'
 
-describe('components | BookingsDateCell', () => {
-  it('should render a div with two span, one containing the date and the other one the time', () => {
+const renderDateCell = props => {
+  return render(<BookingDateCell {...props} />)
+}
+
+describe('bookings | BookingsDateCell', () => {
+  it('should render the date and the time', () => {
```

Comment:
```suggestion
  it('should display the date and the time', () => {
```
Pour être moins technique.
Je te laisse le changer sur tous les autres tests.